internal/integration_test/bench/bench_test.go

```diff
@@ -89,9 +89,10 @@ func runCompilation(b *testing.B, r wazero.Runtime) wazero.CompiledModule {
 func runInitializationBench(b *testing.B, r wazero.Runtime) {
 	compiled := runCompilation(b, r)
 	defer compiled.Close(testCtx)
+	config := wazero.NewModuleConfig().WithSysNanotime().WithSysWalltime().WithRandSource(rand.Reader)
```

Comment:
what's the reason for this change? looks irrelevant and I suppose this will end up introducing non-determinism in the test, which is bad.

Reply:
The reason for moving `wazero.NewModuleConfig()` out here and storing it in a variable to be re-used is because typically you'd want to exclude setup code from your benchmark to limit the measurements to the function under test (in this case `r.InstantiateModule`).

The reason for adding `.WithSysNanotime().WithSysWalltime().WithRandSource(rand.Reader)` is because `platform.NewFakeRandSource()`, `platform.NewFakeWalltime()`, and `platform.NewFakeNanotime()` are all costly to call during each benchmark. I'm just amortizing the config to reduce irrelevant noise in the profile. Because this benchmark is just measuring initialization it didn't seem like loading a real clock and rand source would introduce any non-determinisim but if you prefer it would be just as effective to setup the config with the same fake versions before resetting the timer. Happy to make that change if it's something that's important to you.

Reply:
```suggestion
	// Configure with real sources to avoid performance hit initializing fake ones. These sources are not used
	// in the benchmark.
	config := wazero.NewModuleConfig().WithSysNanotime().WithSysWalltime().WithRandSource(rand.Reader)
```